plugins/trezor/qt_generic.py

```diff
@@ -188,8 +188,6 @@ def create_handler(self, window):
 
     @hook
     def receive_menu(self, menu, addrs, wallet):
-        if type(wallet) is not Standard_Wallet:
-            return
         keystore = wallet.get_keystore()
         if type(keystore) == self.keystore_class and len(addrs) == 1:
```

Comment:
`wallet.get_keystore()` returns the first keystore in the wallet, that is why it only works if Trezor is first